.vscode/settings.json

```diff
@@ -1,16 +1,15 @@
 {
+  "cSpell.words": ["adal", "uninitialize"],
```

Comment:
These are words that won't be marked as misspelled by the spell checking extension.